server/config/routes.json

```diff
@@ -1,14 +1,12 @@
 [
     {"type":"get","uri":"/api/ping","method":"ping","controller":"api"},
  
-    {"type":"post","uri":"/api/webhook","method":"receiveBNPLKYC","controller":"api"},
-    {"type":"post","uri":"/api/bnpl-kyc/webhook","method":"receiveBNPLKYC","controller":"api"},
-    {"type":"post","uri":"/api/bnpl-lender/webhook","method":"receiveBNPLLender","controller":"api"},
-    {"type":"post","uri":"/api/mortgage-borrower/webhook","method":"receiveMortgageBorrower","controller":"api"} ,
-    {"type":"post","uri":"/api/mortgage-lender/webhook","method":"receiveMortgageLender","controller":"api"} 
+    {"type":"post","uri":"/api/webhook","method":"receiveWebhook","controller":"api", "appendParams":{"type":"BNPL_KYC"}},
+    {"type":"post","uri":"/api/bnpl-kyc/webhook","method":"receiveWebhook","controller":"api", "appendParams":{"type":"BNPL_KYC"}},
+    {"type":"post","uri":"/api/bnpl-lender/webhook","method":"receiveWebhook","controller":"api", "appendParams":{"type":"BNPL_Lender"}},
+    {"type":"post","uri":"/api/mortgage-borrower/webhook","method":"receiveWebhook","controller":"api", "appendParams":{"type":"Mortgage_Borrower"}},
+    {"type":"post","uri":"/api/mortgage-lender/webhook","method":"receiveWebhook","controller":"api", "appendParams":{"type":"Mortgage_Lender"}}
```

Comment:
Is this going to break the current deployment when we upgrade the server?

Reply:
This should be in a separate PR

Reply:
no it will not break anything because the routes are not changing

Reply:
The only thing that changes is how the server processes the routes -- and it really isnt changed at all the code just looks different

Reply:
but you are right in that it should have been a diff PR